typescript/src/app.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  authenticate,
+  createOrder,
+  getAccountInfo,
+  getOpenOrders,
+  listAvailableMarkets,
+  onboardUser,
+} from "./utils/api";
+import { Account, SystemConfig } from "./utils/types";
+import { shortString } from "starknet";
+
+// Example usage of the Paradex API
+(async () => {
+  // TODO: Get from /system/config
+  const apiBaseUrl = "https://api.testnet.paradex.trade/v1";
+  const chainId = shortString.encodeShortString("PRIVATE_SN_POTC_SEPOLIA");
+  const config: SystemConfig = {
+    apiBaseUrl,
+    starknet: { chainId },
+  };
+
+  // TODO: Add key derivation
```

Comment:
Do we want to follow up on these TODOs before merging? (In my understanding, we can follow up later 👍)

Reply:
Will be raising follow-up PRs for these